openbook_posts/views/post/views.py

```diff
@@ -201,8 +202,52 @@ def post(self, request, post_uuid):
             return ApiMessageResponse(_('Max length of translation text exceeded.'),
                                       status=status.HTTP_400_BAD_REQUEST)
 
-        post_serializer = TranslatePostResponseSerializer(post, context={'request': request,
-                                                                         'translated_text': translated_text})
+        return Response({'translated_text': translated_text}, status=status.HTTP_200_OK)
 
-        return Response(post_serializer.data, status=status.HTTP_200_OK)
 
+class GetPostParticipants(APIView):
+    permission_classes = (IsAuthenticated,)
+
+    def get(self, request, post_uuid):
+        request_data = request.query_params.dict()
+
+        request_data['post_uuid'] = post_uuid
+
+        serializer = GetPostParticipantsSerializer(data=request_data)
+        serializer.is_valid(raise_exception=True)
+
+        user = request.user
+        data = serializer.validated_data
+
+        post_uuid = data['post_uuid']
+        count = data['count']
+
+        post_participants = user.get_post_with_uuid_participants(post_uuid=post_uuid)[:count]
```

Comment:
perhaps change to  `user.get_participants_for_post_with_uuid`, we have names like this almost everywhere

Reply:
👍